modules/@apostrophecms/file/index.js

```diff
@@ -18,7 +18,8 @@ module.exports = {
     slugPrefix: 'file-',
     autopublish: true,
     editRole: 'editor',
-    publishRole: 'editor'
+    publishRole: 'editor',
+    showPermissions: true
```

Comment:
Replaces `interestingTypes`, allows module developers to say what should happen with their type. In this case, opting in to always being called out separately in the permissions explorer.